lib/api/objectGetACL.js

```diff
@@ -91,6 +91,14 @@ export default function objectGetACL(authInfo, request, log, callback) {
                         { method: 'objectGetACL', error: err });
                         return next(err, bucket);
                     }
+                    if (objectMD.isDeleteMarker) {
+                        log.trace('delete marker detected',
```

Comment:
I think 'most recent version is a delete marker' is clearer for case where return NoSuchKey and a separate log stating 'requested version is a delete marker' when return MethodNotAllowed.